swarm/network/stream/streamer_test.go

```diff
@@ -1265,11 +1322,11 @@ func TestGetSubscriptionsRPC(t *testing.T) {
 				t.Fatal(err)
 			}
 			//length of the subscriptions can not be smaller than number of peers
-			log.Debug(fmt.Sprintf("node %s subscriptions:", node.String()))
+			log.Debug("node subscriptions:", "node", node.String())
```

Comment:
The `String()` method should be called automatically, you don't have to explicitly call it.